docs/2_QUICK_START_GUIDE.md

```diff
@@ -1,13 +1,229 @@
-< Outline the goals and takeaways of this walkthrough/lab >
+
+🔲 Prepare local environment for *Cobalt Infrastructure Template* deployments.
+
+🔲 Deploy the [*Azure Hello World CIT*](../infra/templates/az-hello-world/README.md "AZ Hello World - Cobalt Infrastructure Template").
+
+🔲 Walk away with a introductory understanding of the *CIT* developer workflow.
+
+🔲 Feel confident in moving forward to our next recommended section: *[Cobalt Templating from Scratch](https://github.com/microsoft/cobalt/blob/master/docs/3_NEW_TEMPLATE.md).*
 
 ## 2.3 Prerequisites
-< Enumerate any prerequisite walkthroughs that should be completed prior to this, and any technical prerequisites such as developer environment / tools >
 
-## 2.4 Walkthrough
-< Step-by-step instructions for completing this walkthrough >
\ No newline at end of file
+> NOTE: Previous "infrastructure as code" experience is not a prerequisite for completing the quick start guide.
+
+| Azure Prereqs | Description | I | Install Prereqs | Description |
+|----------|--------------|-|----------|--------------|
+| `Azure Subscription` | [Azure Portal](https://portal.azure.com/) - This template needs to deploy infrastructure within an Azure subscription.|I|`Terminal with bash shell`|[WSL](https://code.visualstudio.com/docs/remote/wsl) or [Git Bash](https://git-scm.com/downloads) - The shell environment needed to follow along with the provided instructions.|
+|`Azure Service Principal`|[Azure Service Principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) - This template needs permissions to deploy infrastructure within an Azure subscription.|I|`Git Source Control`|[Install git](https://www.atlassian.com/git/tutorials/install-git)|
+|`Azure Storage Account`|[Azure Storage Account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview) - An account for tracking terraform remote backend state. You can use our backend state setup [template](../infra/templates/backend-state-setup/README.md) to provision the storage resources.|I|`Terraform`|[Terraform](https://www.terraform.io/downloads.html) - Download the appropriate environment for setting up Terraform|
+|`Azure CLI`|[Get started with Azure CLI](https://docs.microsoft.com/en-us/cli/azure/get-started-with-azure-cli?view=azure-cli-latest)- An account for tracking terraform remote backend state. You can use our backend state setup [template](../infra/templates/backend-state-setup/README.md) to provision the storage resources.|I|`GitHub Account`|[Github](https://github.com/login) - An account for forking the Cobalt repo|
+
+---
+
+## 2.4 Deploy Cobalt's [_Azure Hello World CIT_](../infra/templates/az-hello-world/README.md)
+
+Below are the steps for deploying the [_Azure Hello World CIT_](../infra/templates/az-hello-world/README.md). Ensure that this deployment outputs the following Azure cloud resources before you can call this quick start guide finished:
```

Comment:
> Ensure that this deployment outputs the following Azure cloud resources before you can call this quick start guide finished

If I don't know what a terraform output is, which I may not based on the requisite knowledge called out above, then I don't understand this line. Maybe we can just call this out as a step below and omit it from here.

Reply:
I removed the word "output". It's a reserved term in Terraform as well. Thanks! Here's the new sentence:
> Ensure that running this template deploys the following Azure cloud resources before you can call this quick start guide finished: